### PR TITLE
Fix for windows server and macOS/Unix client

### PR DESCRIPTION
**Pull request by Hackmodford**
This is how I solved #284 

I'm not sure this is the best solution since I am no typescript expert.

### CODE REVIEW

**Comment by earth2016** [comment]
Please, this PR seems interesting to me i would like to have it for testing it.

File: src/paths.ts [hunk 1]
```diff
@@ -45,6 +45,12 @@ export function convertDebuggerPathToClient(
     } else {
         localPath = path.normalize(serverPath)
     }
+        
+    if (serverIsWindows && !isWindowsUri(localPath)) {
```
Comment (felixfbecker) [inline src/paths.ts]: `localPath` is a path, not a URI, so it's not correct to use `isWindowsUri()` on it

Comment (Hackmodford) [reply src/paths.ts]: Thanks for the input. I am not sure how to fix this and mostly submitted this pull request for the discussion. It does solve the issue on my machine, but if anybody knows a better solution, by all means please help :)

**Review by lastthyme — APPROVED** [review]

**Comment by zobo** [comment]
Is this somehow relevant? There were a number of changes in this logic (and some pending ones) and if it's still a problem I'll write tests and see about the change. Honestly I did not use it in this context (server windows, client POSIX).

**Comment by zobo** [comment]
Outdated and obsolete. Using `clientIsWindows ? path.win32 : path.posix` and `serverIsWindows ? path.win32 : path.posix`.

**Pull request closed without merging**